### PR TITLE
have a more explicit relationship to fields in exhibitions

### PR DESCRIPTION
**Pull request by jamesgorrie**
This has not been pushed and can only be once we have done the client work for it.
Basically making the model more explicit after having a sit and think and the chat with Silver.

### CODE REVIEW

File: prismic/custom-types/exhibitions.json [hunk 1]
```diff
@@ -31,14 +45,6 @@
...
       }
     },
-    "gallery_level" : {
-      "type" : "Select",
-      "config" : {
-        "options" : [ "0", "1", "2" ],
-        "label" : "gallery level",
-        "placeholder" : "select gallery level"
-      }
-    },
```
Comment (jamesgorrie) [inline prismic/custom-types/exhibitions.json]: [Now see here](https://github.com/wellcometrust/wellcomecollection.org/pull/1559/files#r140161346)

**Comment by jamesgorrie** [comment]
Related to: https://github.com/wellcometrust/wellcomecollection.org/pull/1559/files

File: prismic/custom-types/exhibitions.json [hunk 2]
```diff
@@ -80,94 +86,6 @@
       }
     }
   },
-  "Body" : {
-    "body" : {
-      "fieldset" : "Body content",
-      "type" : "Slices",
-      "config" : {
-        "choices" : {
-          "standfirst" : {
```
Comment (davidpmccormick) [inline prismic/custom-types/exhibitions.json]: What's the plan for the standfirst-esque thing on the Exhibition template?

Comment (jamesgorrie) [reply prismic/custom-types/exhibitions.json]: [`subtitle`](https://github.com/wellcometrust/wellcomecollection-content-model/pull/13/files#diff-6c448b07db71e18bb1e7f25df20130f9R10).

Comment (jamesgorrie) [reply prismic/custom-types/exhibitions.json]: the fact you used `esque` made me think it's a redundant editorial, we-print-stuff-still-no? mentality.

**Pull request merged**